### PR TITLE
17 populate presentation

### PR DESCRIPTION
**Pull request by nateschor**
In #17 we adjusted the number of rolls in our simulation, the histogram's color, and the size of the axis labels. 

As part of this review:
* @nateschor reviews @kobleary's changes
* @kobleary reviews @nateschor's changes

### CODE REVIEW

File: analysis/source/plot/create_plots.R [hunk 1]
```diff
@@ -23,20 +23,21 @@ main <- function() {
   
 }
 
-plot_hist <- function(data_path, out_path, var_name, in_file, out_file, height = 5, width = 6){
+plot_hist <- function(data_path, out_path, var_name, in_file, out_file, height = 4, width = 5){
```
Comment (nateschor) [inline analysis/source/plot/create_plots.R:26]: @kobleary does `height = 4` and `width = 5` render well in **LaTeX**?

File: analysis/source/simulation/designs/500_8.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
-sample_size,500
+sample_size,1000
```
Comment (kobleary) [inline analysis/source/simulation/designs/500_8.csv:1]: @nateschor Is 1000 enough simulations?

**Review by nateschor — COMMENTED** [review]
@kobleary I just have one question about figure size, otherwise this looks great. Thanks!

**Review by kobleary — COMMENTED** [review]
@nateschor Looks good! I just have a question about the number of simulations.

File: analysis/source/plot/create_plots.R (hunk 1 above)
Comment (kobleary) [reply analysis/source/plot/create_plots.R:26]: @nateschor yep, we've used this sizing for another paper and it worked great.

File: analysis/source/simulation/designs/500_8.csv (hunk 2 above)
Comment (nateschor) [reply analysis/source/simulation/designs/500_8.csv:1]: @kobleary good point. Based on the simulations [here](https://ditraglia.com/Econ103Public/Rtutorials/Rtutorial4.html#approximating-probabilities), the sample distributions are close for $N = 1,000$ and $N = 100,000$. So it looks like $X_n \xrightarrow{\text{d}} X$ where $X = U_1 + U_2$, $U_i \sim \text{Unif}\\{1,8\\}$.

I think 1,000 trials is sufficient, please let me know if you disagree?

Comment (kobleary) [reply analysis/source/simulation/designs/500_8.csv:1]: @nateschor sounds good. resolving this!

**Comment by nateschor** [comment]
@SimonFreyaldenhoven me and @kobleary were thinking of keeping this branch but not merging it into main. It would be nice if there was a way to "close" or "hide" a branch, but since (to my knowledge) there isn't, we thought it would be helpful to keep the branch in case people wanted to follow along or use it as a reference. We can close the PR if you'd like so that it doesn't clutter up the [Open PRs](https://github.com/SimonFreyaldenhoven/example_template/pulls?q=is%3Aopen+is%3Apr).

What do you think?

**Comment by SimonFreyaldenhoven** [comment]
@nateschor works for me! Closing the PR now. Reopening is easy should we ever want to.

**Pull request closed without merging**